node/src/chain_spec.rs

```diff
@@ -232,6 +233,15 @@ pub fn testnet_genesis(
 	para_id: ParaId,
 	chain_id: u64,
 ) -> GenesisConfig {
+	// This is supposed the be the simplest bytecode to revert without returning any data.
+	// We will pre-deploy it under all of our precompiles to ensure they can be called from
+	// within contracts. TODO We should have a test to ensure this is the right bytecode.
+	// (PUSH1 0x00 PUSH1 0x00 REVERT)
```

Comment:
This sounds like a useful test. Something like:
1) modify state
2) revert
3) ensure state wasn't modified

Reply:
Maybe we should make a ticket for this and save it for our next round of `testweek` :)

Reply:
It's added to the top of moonbeam jira 495